e2e/paths/deleteCase_test.js

```diff
@@ -0,0 +1,17 @@
+const config = require('../config.js');
+
+Feature('Submit Case').retry(2);
+Before((I, caseViewPage) => {
+  I.logInAndCreateCase(config.swanseaLocalAuthorityEmailUserOne, config.localAuthorityPassword);
+  I.selectOption(caseViewPage.actionsDropdown, config.applicationActions.deleteCase);
+  I.click(caseViewPage.goButton);
+});
+
+Scenario('Can delete a case and not have actions within it anymore', (I, caseViewPage, deleteCasePage) => {
+  deleteCasePage.delete();
```

Comment:
Please rename page method, you are not deleting anything, you ticking consent box.